modules/graphstudio/pages/write-queries.adoc

```diff
@@ -45,41 +53,32 @@ To edit an existing query, click on the query name in the list in the left panel
 === Save Query Draft
 
 Click the "save" buttonimage:save_btn.png[]in the toolbar to save changes as a draft.
-This is separate from installing a query.
+This is separate from installing a query, allowing you to save your progress if you navigate away from the GraphStudio page.
 
 === Save as
 
 Click the 'Save as' button to save the current query as a new query under a different name. 
 After saving a query as another query, you will be redirected to the new query, and the original current will return to its last saved state.
 
-=== Graph Data Science Library Queries
-
-GraphStudio is connected to the xref:graph-ml:intro:index.adoc[Graph Data Science Library] and can import machine learning algorithms directly from a curated library.
-Click btn:[Choose Query] on the Add New Query window to see the query choices.
-Select any queries that you would like to install.
-After installation, they will appear in the GSQL Queries window just like user-generated queries.
-
-[NOTE]
-If you use the Data Science Library to install a query that is already installed, installation will fail.
-
 === Run Query in Interpreted Mode
 
-You can run a query in interpreted mode if you have saved it without errors.
+You can run a query in interpreted mode if you have saved it as a draft without errors.
 Click the run query button image:start-loading-button.png[].
-Together with the query execution result, a warning message will appear to notify user of the performance concerns running the query in interpreted mode. To speed up the query performance after you are satisfied with the GSQL query developed in interpreted mode, you can follow the instructions in the next section "Install Query".
-
+Together with the query execution result, a warning message will appear to notify you of the performance concerns running the query in interpreted mode.
+To speed up the query performance after you are satisfied with the query, follow the instructions in the next section to install the query.
 
 === Install Query
 
-Saving a query enables the "install query" button image:publish_btn.png[].
+Saving a query enables the "Install Query" button image:publish_btn.png[].
 Click it to install the query, a process which will take about a minute.
+This improves performance compared to running a query in interpreted mode.
 
 image::install_query_progress.png[]
 
 == Run Query
 
 To run the query, click on the "run" button  image:run-installed-query.png[image] in the toolbar.
-If the query has no parameters, it will run directly and the result will be shown in the Result panel.
+If the query has no parameters for you to set, it will run immediately and the result will be shown in the Result panel.
 
```

Comment:
If the query does have input parameters, then the "Enter query parameters" pane will open, on the left side of the query editing sub-panel.  If there are several parameters, you may need to scroll down to see them all. After entering in all the values you wish to use, click the "Run query" button at the bottom of the pane.